vmm/src/lib.rs

```diff
@@ -347,6 +355,17 @@ pub fn start_event_monitor_thread(
                         e
                     })?;
             }
+            if landlock_enable {
+                Landlock::new()
+                    .map_err(Error::ApplyLandlock)?
+                    .restrict_self()
+                    .map_err(Error::ApplyLandlock)
```

Comment:
I think this should be two different errors to help trace them.

Reply:
fixed